### DISCUSSION TITLE
Get to know GitHub Universe!

### DISCUSSION THREAD

**Discussion by tuves** [community/community#63371] [Announcements]
### What is GitHub Universe? 👀

- [GitHub Universe](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=101-intro) is our annual global event designed to bring together the world’s developers, enterprise leaders, and security professionals for two days of networking, connecting with peers and the team at GitHub, and seeing the latest innovations. 
- This year, we’ll have over 80 sessions covering the developer experience, security, and AI. With virtual and in-person ticket options and multiple session formats, you can craft a customized learning experience fit for your personal growth and professional development! 
- You can join us in-person at the Yerba Buena Center for the Arts (YBCA) in San Francisco, or virtually from anywhere on November 8-9.

### Why should I attend? 🤗

- Expand your network. We’re welcoming over 2,000 attendees to San Francisco and tens of thousands more virtually. Whether you’re a CTO, engineering manager, seasoned expert, or just starting out, this is your community. 
- Learn from the best. YBCA will have five stages running concurrently throughout the day with session formats like panels, fireside chats, and more. We’ll also have interactive learning opportunities with product leaders and partners on the Demo Stage. If you can’t make it in-person or stay the whole time, don’t worry: Many sessions will be available on demand shortly after the event.
- Level-up with interactive sessions.  Register for one (or many) of the nine unique workshops and boot camps starting on November 7 and running through November 9. 
- Get certified: Enhance your skill set when you attend in-person by choosing one (or more) certifications in GitHub Actions, GitHub Advanced Security, or GitHub Administration. More information is coming soon!
- Experience new technology. Be the first to hear about GitHub’s latest product announcements and get hands-on demos of new GitHub features.
- Be inspired. YBCA provides a platform for new and established artists to connect with their community. This year, our campus expands to include the San Francisco Museum of Modern Art (SF MoMA), with additional special programming at the Hyatt Regency. 

### When is it? 🗓️

November 8-9, 2023

### Where is it? 🎨

- IRL at the Yerba Buena Center for the Arts in San Francisco. 
- Virtually from wherever you are. (Streaming link coming soon!) 

### When can I buy tickets? 🎟️

Coming soon on August 15! Take advantage of our super-early bird discount of 30% off in-person tickets, only for a limited time. 

### I’m a junior developer, should I attend? 💻
Yes! Developers of all levels are welcome. Our session content will span beginner, intermediate, and advanced levels to accommodate multiple career stages and learning paths. 
### I’m a senior developer, should I attend? 💻
Absolutely! With intermediate and advanced levels, product demos and ample networking opportunities, Universe is geared towards professionals and tech enthusiasts at all stages of their careers. 

### Stay in touch 👋 
Be the first to hear when registration opens on August 15 by [signing up](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=101-intro) for our email list! In-person tickets will be on sale for a limited time.
### Keep up to date 📱 
Follow [@githubuniverse](https://twitter.com/github) on Twitter and @github on LinkedIn, Instagram, and TikTok for the latest.

**Comment by MFasihUddin** [comment]
Awsome! it will really helpfullfor us.

**Reply by AhmadXeeke** [reply]
wow this is awesome

**Comment by mdsifathosen951** [comment]
48435243

**Comment by m0n0t0ny** [comment]
Got my ticket and... already looking forward to it!

**Comment by xieyuheng** [comment]
We need old feed back:

https://github.com/orgs/community/discussions/66244

**Comment by NoeMLinh** [comment]
I'm a student, so at the "Company Information", what should i fill in?

**Comment by donuts-are-good** [comment]
Guys your timezones are broken in the scheduler. You are PST, I am CST. Your schedule shows my timezone, not PST timezone, so it is 2 hours ahead all the time. 

<img width="1285" alt="image" src="https://github.com/community/community/assets/96031819/7d67391e-199f-4a79-9c7f-0a0b8bf8a4a4">


Scheduling (the green line) is 2 hours ahead because the graph is showing me CST not PST

**Comment by NK-debug** [comment]
I wish you used a platform [for GitHub Universe 2023] that allowed rewind on the LiveStream; Microsoft conferences can do that.
Allows one to quickly [and immediately] review a presentation screen/view that changed.
Just my $0.02 🙂

**Comment by es10176** [comment]
is this 18 and over only
